src/vm/perfmap.h

```diff
@@ -19,6 +21,7 @@ class PerfMap
 
     // The file stream to write the map to.
     CFileStream * m_FileStream;
+    PerfInfo* m_PerfInfo;
```

Comment:
Please put a line in between the FilStream and the PerfInfo declaration and put a comment on the PerfInfo declaration.